### ISSUE TITLE
Create a gallery of interesting books

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/meta#154]
We should create a gallery of interesting Jupyter Books that the community can use as inspiration (and to highlight both Jupyter Book and the other projects). A few quick ideas:

- We could take a list of public repositories (and optionally paths to a book within those repositories) in a YAML file
  - For each one, use a `logo` file defined in `_config.yml` to display a logo
  - We also need the URL of the deployed book for that repository
- We have a repository at `executablebooks/gallery` that runs a GitHub action that loops through these books and programmatically builds up a `sphinx-pannels` gallery of links that people can click
- Any new PR that's merged to this repo will auto-update the gallery

- Who decides what books are in the gallery? (a few options)
  - We hand-pick books
  - We ask users to self-select and accept them all
  - We have a publicly-editable list that others can add to
  - We come up with something fancy like "open an issue with a template that asks you to provide information" and we either choose 👍 or 👎 , or ask others to upvote the issue and above some threshold the repository is displayed

# Short-list of books to display

- [Quantitative Economics book by mysteriously unknown but clearly talented authors](https://jstac.github.io/continuous_time_mcs/intro.html)
- [Intro to DS/Stats as part of a course at NYU](https://cranmer.github.io/stats-ds-book/)

**Comment by chrisjsewell** [comment]
yeh I think initially we can hand pick some, to get going, then people can make PRs; updating the list of repositories with their own, and we moderate whether to allow them (generally just a cursory check of the site URL).
Then probably ordering them by the number of stars on the repo would be simplest, and/or you could get more intricate, by e.g. "spotlighting" new repositories added (even if they have no stars), just for a few days or so

**Comment by choldgraf** [comment]
that sounds good to me - do you know of any books out there to hand-pick from? Or shall I try to blast twitter or something?

**Comment by jstac** [comment]
Just stumbled across [this one](https://jstac.github.io/continuous_time_mcs/intro.html), never heard of the authors but it looks like great quality.

Seriously, in the admonition/tip "Get Involved with Jupyter Book", perhaps add an item asking for contributions to a  gallery?

**Comment by choldgraf** [comment]
haha :-)

re: adding it to the front page, I think that's a great idea! Once we figure out a pattern for others to follow then we can share it there.

**Comment by choldgraf** [comment]
another cool one from a friend of mine at NYU: https://cranmer.github.io/stats-ds-book/

**Comment by jmason86** [comment]
Us (@mbobra) too, maybe! [helioml.org](http://helioml.org)

**Comment by choldgraf** [comment]
definitely! (though y'all should upgrade to the latest jupyter book!)

**Comment by jmason86** [comment]
That's the plan. A bunch of stuff broke when we tried last week and we haven't gone back to debug yet.

**Comment by chrisjsewell** [comment]
FYI, this is somewhat equivalent to how we handle plugins in AiiDA: https://github.com/aiidateam/aiida-registry: they make a PR which triggers a number of tests on their repo, to check which version of aiida-core they support etc, then when the PR is merged it triggers the documentation build to include their plugin in the registered list: https://aiidateam.github.io/aiida-registry/

**Comment by choldgraf** [comment]
ah yes that looks fantastic! Basically I think we just want that + a image preview for each book (either a logo or somehow grab a screenshot of their front page)

**Comment by jstac** [comment]
I agree, that looks great.  If the image is hard then I reckon we should just go ahead with a title and authors, setting the issue of images aside for now.

**Comment by choldgraf** [comment]
Title and authors I think should be easy - if there's a logo that shouldn't be too hard either (since a logo path + a URL means we should just be able to link externally for the image)...but a screenshot might be tricky

**Comment by TomasBeuzen** [comment]
A few others I've come across:

- [Python Packaging book: Py-Pkgs](https://py-pkgs.org/) (shameless plug that this is my own but not the latest JB yet - working on it!) 
- [The Climate Laboratory Book](https://brian-rose.github.io/ClimateLaboratoryBook/home.html)
- [Geographic Data Science with PySAL and the PyData Stack](https://geographicdata.science/book/intro)
- [Computational and Inferential Thinking](https://www.inferentialthinking.com/chapters/intro)

**Comment by choldgraf** [comment]
another cool one I just noticed from @melaniewalsh!  https://melaniewalsh.github.io/Intro-Cultural-Analytics

**Comment by bknaepen** [comment]
Our group has put online its first Jupyter Book. It's still work in progress as it's the first year this course is taught and we add material on a bi-weekly basis. Jupyter Book is really convienent to collect our notebooks in an easy to navigate website for the students. Thanks for all the great work !

https://aquaulb.github.io/book_solving_pde_mooc/

**Comment by jstac** [comment]
That PDE book is really nice @bknaepen -- definitely should be in the gallery.

**Comment by thomasfermi** [comment]
Hi everybody. I am just gonna do shameless self-promotion and mention my [Jupyter Book on Algorithms for Automated Driving](https://thomasfermi.github.io/Algorithms-for-Automated-Driving/Introduction/intro.html) !

I really enjoy to use Jupyter Book. The recent addition of tabs was awesome. I used tabs to 
* [let the students choose how many hints they want for an exercise](https://thomasfermi.github.io/Algorithms-for-Automated-Driving/LaneDetection/Segmentation.html#gathering-training-data)
* [let the students see how a figure changes when a parameter is changed](https://thomasfermi.github.io/Algorithms-for-Automated-Driving/Control/PurePursuit.html) (could be done with matplotlib anim I guess, but I generated that figure with tikz)
